### PR TITLE
Integrate old initramfs in the new framework

### PR DESCRIPTION
**Pull request by telphan**
Connects to resin-os/meta-resin#379

This PR is incomplete, because it awaits this (resin-os/meta-resin#594) to be merged.

### CODE REVIEW

File: layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+labelrootfs_enabled() {
+	return 0
+}
+
+labelrootfs_run() {
+    echo "$SCRIPTNAME: Waiting for udev to populate /dev/disk/by-label/$bootparam_LABEL... "
```
Comment (michal-mazurek) [inline layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs]: inconsistent spaces/tabs

File: layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    while :
+    do
+        if ls -A "/dev/disk/by-label/$bootparam_LABEL" >/dev/null 2>&1; then
```
Comment (michal-mazurek) [inline layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs]: Maybe "test -e" would be more logical?

File: layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs (hunk 1 above)
Comment (telphan) [inline layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs]: My bad.

File: layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs (hunk 2 above)
Comment (telphan) [inline layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs]: It definitely would.

File: layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+labelrootfs_run() {
+    echo "Waiting for udev to populate /dev/disk/by-label/$bootparam_LABEL... "
+    while :
+    do
+        if test -e /dev/disk/by-label/$bootparam_LABEL; then
+        break
+    fi
+    sleep 1
```
Comment (michal-mazurek) [inline layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs]: Is this indented correctly?

File: layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    return 0
+}
+
+labelrootfs_run() {
+    echo "Waiting for udev to populate /dev/disk/by-label/$bootparam_LABEL... "
+    while :
+    do
+        if test -e /dev/disk/by-label/$bootparam_LABEL; then
+            break
+       fi
```
Comment (michal-mazurek) [inline layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs]: Indentation - looks like 3 spaces, not 4.

File: layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    sleep 1
```
Comment (michal-mazurek) [inline layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs]: Indentation, misaligned sleep

File: layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs (hunk 4 above)
Comment (telphan) [inline layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs]: ack

File: layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs (hunk 5 above)
Comment (telphan) [inline layers/meta-resin-intel/recipes-core/initrdscripts/files/labelrootfs]: ack

**Comment by floion** [comment]
Guess this can be merged now

**Comment by agherzan** [comment]
Did anyone test this PR on a board confirming that everything works especially related to initramfs?

**Comment by floion** [comment]
I haven't

**Comment by agherzan** [comment]
@telphan Any feedback?

**Comment by floion** [comment]
Okay, so as it stands now it looks like I will test this PR tomorrow on a board and let's have this merged if testing is okay because this PR not being merged makes other PRs fail

**Comment by agherzan** [comment]
Thanks man. I appreciate.

**Comment by telphan** [comment]
@floion Thank you for taking up testing on this.

**Pull request merged**